tests/foreman/cli/test_repository.py

```diff
@@ -714,21 +723,22 @@ def test_positive_synchronize_file_repo(self, repo_options, repo, target_sat):
     @pytest.mark.tier2
     @pytest.mark.parametrize(
         'repo_options',
-        **parametrized(
-            [
-                (
-                    {
-                        'content-type': 'yum',
-                        'url': FAKE_5_YUM_REPO,
-                        'upstream-username': creds['login'],
-                        'upstream-password': creds['pass'],
-                    }
-                    for creds in valid_http_credentials()
-                    if not creds['http_valid']
-                )
-            ]
-        ),
+        [
+            {
+                'content-type': 'yum',
+                'url': FAKE_5_YUM_REPO,
+                'upstream-username': creds['login'],
+                'upstream-password': creds['pass'],
+            }
+            for creds in valid_http_credentials()
+            if not creds['http_valid']
+            and len(creds['login']) < 255
+            and not (not creds['login'] and creds['pass'])
+            and can_encode_latin(creds['login'])
+            and can_encode_latin(creds['pass'])
+        ],
         indirect=['repo_options'],
+        ids=lambda x: x.get("upstream-username") + ":" + x.get("upstream-password"),
```

Comment:
I would rather use something static for `ids` and log the username and password instead (for case of failure).
The reason behind is that dynamic `ids` would break the continuity in our reporting tools - this would be considered as a different test in each run.

WDYT @SatelliteQE/tier-2-reviewers ?

Reply:
Shame ... but I get it. In that case I can also just delete this line. I do not require logging, debugging is already over ;)